spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -127,6 +127,7 @@ object SpdxDeclaredLicenseMapping {
         "BSD style license" to BSD_3_CLAUSE.toExpression(),
         "BSD*" to BSD_3_CLAUSE.toExpression(),
         "BSD-like license" to BSD_3_CLAUSE.toExpression(),
+        "BSD-Style" to BSD_3_CLAUSE_ATTRIBUTION.toExpression(),
```

Comment:
Please use a permalink that cannot break in the commit message:

https://github.com/jsuereth/scala-arm/blob/366e8d9/build.sbt#L68

Also, I guess this should make to plain `BSD_3_CLAUSE` (not attribution).

Reply:
Fixed